package.json

```diff
@@ -16,12 +16,16 @@
 		"@playwright/test": "^1.28.1",
 		"@sveltejs/adapter-auto": "^2.0.0",
 		"@sveltejs/kit": "^1.5.0",
+		"@tailwindcss/line-clamp": "^0.4.4",
```

Comment:
for 99% this library is not needed :)
![image](https://user-images.githubusercontent.com/26543876/234617615-ab47e8a3-53a7-4298-b504-efe79edf01f8.png)